build/dappnode/scripts/dappnode_install.sh

```diff
@@ -194,13 +203,27 @@ dappnode_start() {
     sed -i '/return/d' $DAPPNODE_PROFILE | tee -a $LOGFILE
 
     if ! grep -q 'http://my.dappnode/' "$DAPPNODE_PROFILE"; then
-        echo "echo -e \"\nTo get a VPN profile file and connect to your DAppNode, run the following command:\"" >>$DAPPNODE_PROFILE
-        echo "echo -e \"\n\e[32mdappnode_connect\e[0m\"" >>$DAPPNODE_PROFILE
-        echo "echo -e \"\nTo connect to your dappnode via Wifi use the following credentials:\"" >>$DAPPNODE_PROFILE
-        echo "echo -e \"\n$(cat /usr/src/dappnode/DNCORE/docker-compose-wifi.yml | grep 'SSID\|WPA_PASSPHRASE')\"" >>$DAPPNODE_PROFILE
-        echo "echo -e \"\nOnce connected through the Wifi or VPN (OpenVPN) you can access to the admin UI by following this link:\"" >>$DAPPNODE_PROFILE
-        echo "echo -e \"\nhttp://my.dappnode/\n\"" >>$DAPPNODE_PROFILE
+        if $MODE=="VPN" ; then
+            echo "echo -e \"\nTo connect to your dappnode via Wifi use the following credentials:\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"\n$(cat /usr/src/dappnode/DNCORE/docker-compose-wifi.yml | grep 'SSID\|WPA_PASSPHRASE')\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"\nTo get a VPN profile file and connect to your DAppNode, run the following command:\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"\n\e[32mdappnode_connect\e[0m\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"\nOnce connected through the Wifi or VPN (OpenVPN) you can access to the admin UI by following this link:\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"\nhttp://my.dappnode/\n\"" >>$DAPPNODE_PROFILE
+            echo -e "return\n" >>$DAPPNODE_PROFILE
+        else
+            echo "echo -e \"\nTo connect to your dappnode via Wifi use the following credentials:\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"\n$(cat /usr/src/dappnode/DNCORE/docker-compose-wifi.yml | grep 'SSID\|WPA_PASSPHRASE')\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"\nYour Wireguard configuration file is as follows\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"=================================================\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"$(eval ${CRED_CMD})\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"=================================================\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"\nInstructions for setting up a Wireguard client are available at https://xxxx.dappnode.io\"" >>$DAPPNODE_PROFILE
+            echo "echo -e \"\nIf your router doesn't support NAT Loopback and you want to access your DAppNode from local network via Wireguard add additional configuration with Endpoint field changed to DAppNode's local IP\"" >>$DAPPNODE_PROFILE
```

Comment:
I think is too much text for the user, what about just displaying credentials and endpoints

Reply:
This is just sketch what it is supposed to be, nothing final.